### PR TITLE
Rename GDPR UserSyncIfAmbiguous to DefaultValue

### PR DESCRIPTION
**Pull request by bsardo**
Addresses the first part of #1783

### CODE REVIEW

**Review by hhhjort — APPROVED** [review]
Looks good. A nice simple, yet comprehensive change.

File: config/config.go [hunk 1]
```diff
@@ -193,7 +193,7 @@ type Privacy struct {
 type GDPR struct {
 	Enabled                 bool         `mapstructure:"enabled"`
 	HostVendorID            int          `mapstructure:"host_vendor_id"`
-	UsersyncIfAmbiguous     bool         `mapstructure:"usersync_if_ambiguous"`
+	DefaultValue            bool         `mapstructure:"default_value"`
```
Comment (SyntaxNode) [inline config/config.go:196]: Let's make this a string to match PBS-Java.

**Review by hhhjort — APPROVED** [review]

File: exchange/exchange.go [hunk 2]
```diff
@@ -51,18 +51,18 @@ type IdFetcher interface {
...
-	bidIDGenerator      BidIDGenerator
+	adapterMap        map[openrtb_ext.BidderName]adaptedBidder
+	bidderInfo        config.BidderInfos
+	me                metrics.MetricsEngine
+	cache             prebid_cache_client.Client
+	cacheTime         time.Duration
+	gDPR              gdpr.Permissions
+	currencyConverter *currency.RateConverter
+	externalURL       string
+	GDPRDefaultValue  string
```
Comment (guscarreon) [inline exchange/exchange.go:62]: Is it possible to consolidate all GDPR info into one field?

**Review by guscarreon — COMMENTED** [review]
LGTM. Just left a small nitpick

File: exchange/exchange.go (hunk 2 above)
Comment (bsardo) [reply exchange/exchange.go:62]: It's possible but I'd rather not tackle that at this time if that's ok with you. I'll be refactoring the `gdpr.Permissions` object and possibly the `privacyConfig` object shortly which might be a more sensible time to consider this change IMO.

Comment (SyntaxNode) [inline exchange/exchange.go:62]: Nitpick: Can you name this `gdprDefaultValue` to match the casing/scope of the other variables?

**Review by guscarreon — DISMISSED** [review]
LGTM

**Review by SyntaxNode — APPROVED** [review]

**Review by hhhjort — APPROVED** [review]

**Review by hhhjort — APPROVED** [review]
LGTM

**Review by guscarreon — APPROVED** [review]
LGTM

**Pull request merged**